proto/poktroll/gateway/event.proto

```diff
@@ -7,7 +7,12 @@ option (gogoproto.stable_marshaler_all) = true;
 import "cosmos_proto/cosmos.proto";
 import "gogoproto/gogo.proto";
 
-// GatewayStaked defines the event emitted when a gateway has been unstaked.
+// GatewayStaked is emitted when a gateway is staked or up-staked.
+message EventGatewayStaked {
+  string address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"]; // The Bech32 address of the gateway.
```

Comment:
Does it make sense to emit the whole gateway type? I think informing about its stake is important too.